packages/maskbook/src/locales/en-US.json

```diff
@@ -805,5 +806,43 @@
     "plugin_good_ghosting_view_on_explorer": "View on Explorer",
     "plugin_good_ghosting_checking_balance": "Checking Balance...",
     "plugin_good_ghosting_insufficient_balance": "You need at least {{amount}} {{token}} in your wallet to do this.",
-    "plugin_good_ghosting_balance_error": "Failed to check balance. Click to retry."
+    "plugin_good_ghosting_balance_error": "Failed to check balance. Click to retry.",
+    "popups_welcome": "Welcome",
+    "popups_wallet_token": "Token",
+    "popups_wallet_go_back": "Go back",
+    "popups_wallet_start_up_tip": "Connect to your Wallet，Create a new wallet or recover an existing wallet using a seed phrase.",
+    "popups_wallet_password_uppercase_tip": "Must contain an uppercase character",
+    "popups_wallet_password_lowercase_tip": "Must contain a lowercase character",
+    "popups_wallet_password_number_tip": "Must contain a number",
+    "popups_wallet_password_special_character_tip": "Must contain a special character",
+    "popups_wallet_password_dont_match": "Passwords don't match",
+    "popups_wallet_name_placeholder": "Enter 1-12 characters",
+    "popups_wallet_payment_password": "Payment Password",
+    "popups_wallet_tab_assets": "Assets",
+    "popups_wallet_tab_activity": "Activity",
+    "popups_wallet_derivation_path": "Derivation path ({{ path }})",
+    "popups_wallet_previous": "Previous",
+    "popups_wallet_next": "Next",
+    "popups_wallet_loading_token_list": "Loading token lists",
+    "popups_wallet_loading_token_assets": "Loading token assets",
+    "popups_wallet_no_token_found": "No token found",
+    "popups_wallet_backup_wallet": "Back up the wallet",
+    "popups_wallet_backup_json_file": "Json File",
+    "popups_wallet_backup_private_key": "Private Key",
+    "popups_wallet_backup_json_file_confirm_password_tip": "This file has been encrypted and stored with your current password. Your current password is needed to decrypt this file when using it to import wallet.",
+    "popups_wallet_backup_private_key_tip": "Please don’t show anyone your private key. The private key can be used in any wallet that supports ETH without decryption.",
```

Comment:
We support all EVM-compatible chains. So better not use fixed ETH in statements.